control/stabilisation_copter.h

```diff
@@ -73,6 +73,7 @@ typedef struct
 	stabiliser_t attitude_stabiliser;							///< The attitude controller structure
 	stabiliser_t velocity_stabiliser;							///< The velocity controller structure
 	stabiliser_t position_stabiliser;							///< The position controller structure
+	stabiliser_t gimbal_attitude_stabiliser;					///< The gimbal attitude controller structure
```

Comment:
Same thing as above, not everyone will use a gimbal, so it is better not to add it to the stabilisation stack.